pypi_count/command_line_interface.py

```diff
@@ -25,9 +25,12 @@ def main(  # pylint: disable=R0913
     function_with_docstrings: bool = typer.Option(False, "--function_with_docstrings"),
     class_with_docstrings: bool = typer.Option(False, "--class_with_docstrings"),
     class_without_docstrings: bool = typer.Option(False, "--class_without_docstrings"),
+    find_parameters: bool = typer.Option(False, "--find_parameters"),
+    assignment_statements: bool = typer.Option(False, "--assignment_statements"),
+
```

Comment:
These additions to the CLI are not reflected in the `README`. It may be helpful to just remove the entire help message from the README so that your README doesn't become outdated as soon as new options are added in the future.

Reply:
@solisa986